<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiConnect</class>
 <widget class="QWidget" name="UiConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_menu_connect_">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>478</width>
     <height>272</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="title">
    <string>Меню для подключения</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_name_host_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>101</horstretch>
        <verstretch>9</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit:hover { background-color: rgba(144, 238, 144, 0.5); }
QLineEdit {Font-family: Arial;  font-size: 13px;   }</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_user_password_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>101</horstretch>
        <verstretch>9</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit:hover { background-color: rgba(144, 238, 144, 0.5); }
QLineEdit {Font-family: Arial;  font-size: 13px;   }</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_name_host_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Имя хоста</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_name_bd_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>101</horstretch>
        <verstretch>9</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit:hover { background-color: rgba(144, 238, 144, 0.5); }
QLineEdit {Font-family: Arial;  font-size: 13px;   }</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_user_password_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_name_bd_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Имя базы данных</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_port_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>101</horstretch>
        <verstretch>9</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit:hover { background-color: rgba(144, 238, 144, 0.5); }
QLineEdit {Font-family: Arial;  font-size: 13px;   }</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label__port_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_user_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>101</horstretch>
        <verstretch>9</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit:hover { background-color: rgba(144, 238, 144, 0.5); }
QLineEdit {Font-family: Arial;  font-size: 13px;   }</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_user_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>291</y>
     <width>481</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_clear_">
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color: rgb(60, 179, 113);
 color: rgb(255, 255, 255);
border: 2px solid rgb(144, 238, 144);
 border-radius: 5px;
padding: 5px; }

QPushButton:hover {
 background-color: rgb(50, 205, 50);}
                                           </string>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_connect_">
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color: rgb(60, 179, 113);
 color: rgb(255, 255, 255);
border: 2px solid rgb(144, 238, 144);
 border-radius: 5px;
padding: 5px; }

QPushButton:hover {
 background-color: rgb(50, 205, 50);}
                                           </string>
      </property>
      <property name="text">
       <string>Подключится</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>359</y>
     <width>481</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_status_connect_">
      <property name="text">
       <string>Отключено</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_config_">
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color: rgb(60, 179, 113);
 color: rgb(255, 255, 255);
border: 2px solid rgb(144, 238, 144);
 border-radius: 5px;
padding: 5px; }

QPushButton:hover {
 background-color: rgb(50, 205, 50);}
                                           </string>
      </property>
      <property name="text">
       <string>Загрузить базовый конфиг</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_cascade_delete_">
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color: rgb(60, 179, 113);
 color: rgb(255, 255, 255);
border: 2px solid rgb(144, 238, 144);
 border-radius: 5px;
padding: 5px; }

QPushButton:hover {
 background-color: rgb(50, 205, 50);}
                                           </string>
      </property>
      <property name="text">
       <string>Очистить данные </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
